src/components/CertificationMetadataForm/index.tsx

```diff
@@ -58,6 +58,7 @@ const CertificationMetadataForm = (props: Props) => {
         ...request,
         certificationLevel: !props.isReviewCertification ? parseInt(request.certificationLevel) : undefined,
         report: !props.isReviewCertification ? request.report.map(report => report.value) : undefined,
+        subject: props.isReviewCertification && profile?.dapp?.subject ? undefined : request.subject
```

Comment:
In some places, we have `null` for empty object properties and here we have `undefined`. Is there a specific reason for that?

Reply:
This is just to indicate that the properties with undefined, need not be present in the UI to accept values from the user. So, if _isReviewCertification_, then certificationLevel, report and subject will not be present.

Reply:
This was done with the intent to not include the property in the request. I've updated it to a different piece of code now